test/claim-aud.test.js

```diff
@@ -6,7 +6,7 @@ const util = require('util');
 const testUtils = require('./test-utils');
 
 function signWithAudience(audience, payload, callback) {
-  const options = {algorithm: 'none'};
+  const options = {algorithm: 'HS256'};
```

Comment:
We may be able to revert all these test changes with the latest change to `sign`. What do you think?

Reply:
We would be able to revert but the places that we use none and verify we would need to configure that algorithm explicitly, do you see an advantage in using `none` in these tests or was comment more about making the PR as small as possible?

I think I prefer using `HS256` for example rather than `none`, since its something we don't want to encourage and should be used in exceptional cases. Also means in the tests we just configure the algorithm, the secret and no options for the verify - the alternative seems verbose and repetitive IMO.

Let me know what you think :)

Reply:
Yep agreed :)